src/core/services/theming/theming.spec.js

```diff
@@ -880,3 +880,89 @@ describe('md-themable directive', function() {
     expect(el.hasClass('md-default-theme')).toBe(true);
   }));
 });
+
+describe('$mdThemeProvider with custom styles that include nested rules', function() {
+  it('appends the custom styles taking into account nesting', function() {
+    module('material.core', function($mdThemingProvider) {
+      $mdThemingProvider.generateThemesOnDemand(false);
+      var styles =
+          "@media (min-width: 0) and (max-width: 700px) {\n\n"
```

Comment:
Literal strings should be in single quotes ( `'` ) to keep coding style consistency.